### PR TITLE
feat: enable custom theme to be provided to iframe embed

### PR DESCRIPTION
**Pull request by venarius**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This adds the ability to provide a custom theme for the iFrame embed

  I have also written out the complete options for the theme into the docs so it's easier for users to understand which options are possible

- **What is the new behavior (if this is a feature change)?**
fixes https://github.com/novuhq/novu/issues/1041
Screenshot 2022-08-17 033429
Screenshot 2022-08-17 033442
Screenshot 2022-08-17 040442

### CODE REVIEW

**Comment by luizgununes** [comment]
Nice! I'm excited for this feature. A parameter to change the "Notifications" text would be great too.

**Comment by venarius** [comment]
cc @scopsy just for your info: It seems that the widget cypress run is still failing on the notification-list part, still after https://github.com/novuhq/novu/commit/d75d0830afbffbdd01c0e45703cac193d721c4c3

**Comment by scopsy** [comment]
Yes :( So strange, taking a look at it soon.

File: apps/widget/cypress/e2e/branding.spec.ts [hunk 1]
```diff
@@ -37,3 +37,41 @@ describe('App Branding', function () {
...
+      .then((session: any) => {
+        cy.wait(500);
+
+        return cy.task('createNotifications', {
+          identifier: session.templates[0].triggers[0].identifier,
+          token: session.token,
+          subscriberId: session.subscriber.subscriberId,
+          count: 5,
+        });
+      });
```
Comment (scopsy) [inline apps/widget/cypress/e2e/branding.spec.ts:64]: Love the added tests here, you are AWESOME! :)

**Review by scopsy — APPROVED** [review]
Looks great and test passing, created another ticket for the flaky test :|

**Pull request merged**